src/audio/gain_node.rs

```diff
@@ -56,11 +59,6 @@ impl AudioNodeEngine for GainNode {
         }
         inputs
     }
-    fn message(&mut self, msg: AudioNodeMessage, sample_rate: f32) {
-        match msg {
-            AudioNodeMessage::SetAudioParamEvent(event) => {
-                self.gain.insert_event(event.to_event(sample_rate))
```

Comment:
This reverts the AudioParam changes for gainnode

Reply:
They are moved [here](https://github.com/servo/media/pull/31/files/3748b6b36b0ee84ed1dc7ba7e0655e41b84d88f2#diff-ab2867bdbfa223725105fd3200838b45R36)

Reply:
Oh, I see what you did.

(sorry, reviewing on a phone)

I guess this is ok. A lot of the messages are common between nodes so I thought we'd have common messages. But this works.